Rakefile

```diff
@@ -97,3 +97,13 @@ task :publish, [:no_commit_msg] => [:remove_tmp_dir, :remove_output_dir, :build]
     system 'git checkout master'
   end
 end
+
+desc "Generate JSON from documentation"
+task :generate_json_from_documentation
+Dir[File.join(File.dirname(__FILE__), 'lib', 'responses', '*.rb')].each { |file| load file }
+FileUtils.mkdir_p(File.join(File.dirname(__FILE__), 'json-dump'))
+GitHub::Resources::Responses.constants.each { |constant|
+  File.open('json-dump/' + constant.to_s + '.json', 'w') { |file|
```

Comment:
Yeah the constants are symbols, without ruby will throw a fit about no implicit conversion between symbol and string.